services/storage/src/lib.rs

```diff
@@ -11,18 +11,33 @@ Each user can have multiple notifications enabled. These notifications must be p
 from the `Notifications` enum. (There cannot be duplicates)
 
 */
+pub mod users;
 
 pub fn initialize_db() -> Result<()> {
-	let conn = Connection::open("cats.db")?;
+	let conn = Connection::open("users.db")?;
 
 	conn.execute(
-		"create table if not exists cats (
-             id integer primary key,
-             name text not null,
-             color_id integer not null references cat_colors(id)
+		"CREATE TABLE IF NOT EXISTS users (
+			id INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+			tg_handle TEXT,
+			email TEXT,
+			notifier TEXT CHECK (
+				notifier IN ('email', 'telegram') 
+				OR notifier IS NULL
```

Comment:
For what it's worth, we could then remove the notifier from the table. If the `email` is set, send an email; if the `tg_handle` is set, we will send a TG message as well. If a user doesn't want to receive notifications through one of the channels, they can simply remove their email or TG handle.

Reply:
And if we keep the notifier, rather than null meaning send to both, it should indicate that the user is created, they configured their notifications, but still haven't set the notifier.

Reply:
What should we do for such users when notifications are set? Does that just remove them from the notifications schedule _(they don't get any notifications)_?

Reply:
I'm not sure if I interpreted your question correctly, but here's what I had in mind(I'm not saying that all of this should be implemented in this PR, but rather I'm explaining how I envision all of this to work):

The tracker module keeps track of certain events. When an event occurs, we query all the users who have enabled notifications and send the notification through their selected channel (email or Telegram). If the notifier is null, no notification is sent. We could also filter out the users whose notifier is null in the query to avoid iterating over them.